security/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.devonfw.tools.IDEasy.dev</groupId>
+    <artifactId>ide</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>ide-security</artifactId>
+
+  <properties>
+    <maven.compiler.source>21</maven.compiler.source>
+    <maven.compiler.target>21</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.owasp</groupId>
+      <artifactId>dependency-check-core</artifactId>
+      <version>12.1.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.devonfw.tools.IDEasy</groupId>
+      <artifactId>url-updater</artifactId>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.devonfw.tools.IDEasy</groupId>
+      <artifactId>url-updater</artifactId>
+      <version>2025.04.001-SNAPSHOT</version>
+      <scope>compile</scope>
```

Comment:
We introduced `<dependencyManagement>` to our parent `pom.xml`.
All versions should be defined centrally there and other `pom.xml` files should not specify versions.
https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management
https://github.com/devonfw/scm/blob/master/documentation/build/maven-dependency-management.asciidoc

However, first focus on the other issues. We could also merge this and someone else (maven expert) could later create another PR to clean this up...

Reply:
Thanks for the rework to use `${project.version}`.
The rest could also be tidied up after the merge (remove `compile`, declare versions in `<dependencyManagement>` and omit here).